Assets/LeapMotion/Core/Scripts/Utils/Utils.cs

```diff
@@ -764,6 +843,118 @@ public struct ChildrenEnumerator : IEnumerator<Transform> {
 
     #endregion
 
+    #region Quaternion Utils
+
+    /// <summary>
+    /// Converts the quaternion into an axis and an angle and returns the vector
+    /// axis * angle.
+    /// </summary>
+    public static Vector3 ToAngleAxisVector(this Quaternion q) {
+      float angle;
+      Vector3 axis;
+      q.ToAngleAxis(out angle, out axis);
+      return axis * angle;
+    }
+
+    /// <summary>
+    /// Returns a Quaternion described by the provided angle axis vector.
+    /// </summary>
+    public static Quaternion QuaternionFromAngleAxisVector(Vector3 angleAxisVector) {
+      if (angleAxisVector == Vector3.zero) return Quaternion.identity;
+      return Quaternion.AngleAxis(angleAxisVector.magnitude, angleAxisVector.normalized);
```

Comment:
would be nice to calculate the magnitude beforehand so that you can use it to normalize the vector for less cost.  Also might want to check if the magnitude of the axis matters at all, it might not, in which case you could skip the normalization step altogether.

Also like above would be good to specify if this method expects radians or degrees.

Reply:
Quaternion.AngleAxis will automatically normalize the provided axis vector anyway, so I simplified this to just not normalize the vector (so we don't double-calculate magnitude more than the AngleAxis function already would).

Clarified comment to expect angle measure in degrees.